feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -7,11 +7,12 @@ import com.revenuecat.purchases.models.Price
 import com.revenuecat.purchases.models.StoreProduct
 
 // In-apps don't have base plan nor offers
-fun ProductDetails.toStoreProduct(): StoreProduct = this.toStoreProduct(null, emptyList())
+fun ProductDetails.toStoreProduct(): StoreProduct = this.toStoreProduct(null, emptyList(), null)
```

Comment:
could this one be private? also wonder if we should rename this to `toInAppStoreProduct` so that we don't accidentally call it for a subscription

Reply:
Both this one and the one below could be private but are used in tests... It's another reason why I think moving this to a factory would make more sense, so it can be mocked more easily.

About the rename, I think it makes sense yeah, will rename it.